whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -191,7 +191,8 @@ public static Message cantRemoveOrgAttribute() {
     }
 
     public static Message cantCreateShortFormatAsName() {
-        return new Message(Type.ERROR, "Cannot create AS-SET object with a short format name.");
+        return new Message(Type.ERROR, "Cannot create AS-SET object with a short format name. Only hierarchical " +
+                "AS-SET creation is allowed. Refer to the documentation for more information");
```

Comment:
I suggest not writing `Refer to the documentation...` because the error message should be self-explanatory. Maybe explain as `Only hierarchical AS-SET creation is allowed, i.e. at least one ASN must be referenced`